cfme/tests/infrastructure/test_cloud_init_provisioning.py

```diff
@@ -57,9 +59,10 @@ def test_provision_cloud_init(appliance, setup_provider, provider, setup_ci_temp
             'host_name': {'name': host},
             'datastore_name': {'name': datastore}},
         'network': {
-            'vlan': vlan},
+            'vlan': partial_match(vlan)},
```

Comment:
What is `partial_match` needed for here?

Reply:
@bsquizz That's because the actual vlan name is 'ovirtmgmt' but provisioning dialog displays it as 'ovirtmgmgt (ovirtmgmt)' . The displayed name is  formatted as '<vlan> (<vNIC>)' if I remember correctly.